paddle/math/MathFunctions.cpp

```diff
@@ -180,7 +180,6 @@ int getri<double>(const CBLAS_ORDER order,
                   const int lda,
                   const int* ipiv) {
   return dynload::PADDLE_DGETRI(order, N, A, lda, ipiv);
-  return 0;
```

Comment:
It is not related to Android. I found this redundant statement accidentally when I was doing this work. Should I do this change in another pull request?